content/en/agent/autodiscovery/_index.md

```diff
@@ -22,69 +22,51 @@ further_reading:
+**For example**, imagine that you have deployed a simple guestbook application on Docker. It stores each guestbook entry in a Redis database. You are using a container orchestrator for scheduling, so you do not know the host IP address or port number to use for the Datadog Agent to connect to the Redis service. You enable Autodiscovery, and you create a configuration template for Redis that uses `%%host%%` and `%%port%%` variables. Upon container creation, the Datadog Agent retrieves the actual host IP and port number from the Docker API. It generates a static check configuration and begins to gather Redis metrics for your application.
 
-**With Autodiscovery enabled, the Agent runs checks differently.**
 
-The overall process of Datadog Agent Autodiscovery is:
+## Setup
 
-1. **Create and Load Integration template**: When the Agent starts with Autodiscovery enabled, it loads integration templates from all [available template sources][2]; along with the [Autodiscovery container identifiers][3]. Static configuration files aren't suitable for checks that collect data from ever-changing network endpoints like host or ports, so Autodiscovery uses [**Template Variables**][4] for integration template configuration. Those integration template configurations can be loaded into the Agent in 5 main ways:
+Setting up Autodiscovery for your infrastructure requires the following two steps:
 
-  * [Using auto-configuration file shipped with the Agent][5]
-  * [Using a configuration file mounted within the Agent][6]
-  * [Using Key-Value Store][7]
-  * [Using Kubernetes Annotations][8]
-  * [Using Docker Labels][9]
+1. [Enable Autodiscovery](#enable-autodiscovery) for your Datadog Agent.
+2. Create [integration-specific configuration templates](#integration-templates) for each service you wish to monitor. Note that Datadog provides auto-configuration templates for [some common containerized services][3], including Apache and Redis.
 
-2. **Apply an integration template to a specific container**: Unlike in a traditional Agent setup, the Agent doesn't run all checks all the time; it decides which checks to enable by inspecting all containers running on the same host as the Agent and the corresponding loaded integration templates. The Agent then watches for Kubernetes/Docker events&mdash;container creation, destruction, starts, and stops&mdash;and enables, disables, and regenerates static check configurations on such events. As the Agent inspects each running container, it checks if the container matches any of the [Autodiscovery container identifiers][3] from any loaded integration templates. For each match, the Agent generates a static check configuration by substituting the [Template Variables][10] with the matching container's specific values. Then it enables the check using the static configuration.
+### Enable Autodiscovery 
 
-## How to set it up
```

Comment:
Can we focus on showing examples of autodiscovery annotations for a K8 pod/deployment or ECS Task first? The example should explain how to use autodiscovery and how the parameters work

Reply:
I think putting an example of that so far up on the page (before we talk about enabling autodiscovery) is confusing, and would be more helpful added later down on the page.